.github/workflows/build-deploy-dev.yml

```diff
@@ -125,23 +125,22 @@ jobs:
       - name: Log in to OpenShift
         run: |
           oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
-      # Build the images
-      - name: Build SIMS-API Image
+          if [ $? -ne 0 ]; then
+            echo "Failed to log in to OpenShift. Please check the credentials and connection details."
+            exit 1
+          fi
+       # Build the images in parallel.
+      - name: Build db-migrations, api, workers, queues and web Images in parallel
         working-directory: "./devops"
         run: |
-          make oc-build-api
-      - name: Build Workers Image
-        working-directory: "./devops"
-        run: |
-          make oc-build-workers
-      - name: Build Queue Consumers Image
-        working-directory: "./devops"
-        run: |
-          make oc-build-queue-consumers
-      - name: Build Web/Frontend Image
-        working-directory: "./devops"
+            make oc-build-db-migrations &
+            make oc-build-api &
+            make oc-build-workers &
+            make oc-build-queue-consumers &
+            make oc-build-web
+      - name: Wait for parallel builds to finish
         run: |
-          make oc-build-web
+          wait
```

Comment:
for my info, what happens here

Reply:
It just waits until all the parallel process is successful